bundles/jsch-agentproxy-jsch/pom.xml

```diff
@@ -82,7 +82,8 @@ limitations under the License.
             </goals>
             <configuration>
               <!-- the RAT plugin complains if we use the default location and we don't want it in the SCM -->
-              <dependencyReducedPomLocation>${project.build.directory}/dependency-reduced-pom.xml</dependencyReducedPomLocation>
+              <!--<dependencyReducedPomLocation>${project.build.directory}/dependency-reduced-pom.xml</dependencyReducedPomLocation>-->
```

Comment:
Why has this been changed? We might need this if we are shading dependencies, to let the right dependencies be resolved by the Maven reactor.